### PR TITLE
  [Reference] Add missing option delivery_whitelist

### PR DESCRIPTION
**Pull request by TerjeBr**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | SwiftMailerBundle 2.1 |
| Fixed tickets | #4925 |

### CODE REVIEW

**Comment by TerjeBr** [comment]
Added this PR since @e-weimann just closed his PR #4925 without it being merged.

File: reference/configuration/swiftmailer.rst [hunk 1]
```diff
@@ -156,6 +157,17 @@ emails sent during development go to a single account.
 This uses ``Swift_Plugins_RedirectingPlugin``. Original recipients are available
 on the ``X-Swift-To``, ``X-Swift-Cc`` and ``X-Swift-Bcc`` headers.
 
+delivery_whitelist
++~~~~~~~~~~~~~~~~~~
```
Comment (xabbuh) [inline reference/configuration/swiftmailer.rst]: You have to remove the plus characters you copied from the old PR. :)

Comment (TerjeBr) [inline reference/configuration/swiftmailer.rst]: Yes, forgot that. Done that now.

File: reference/configuration/swiftmailer.rst [hunk 2]
```diff
@@ -156,6 +157,17 @@ emails sent during development go to a single account.
...
 on the ``X-Swift-To``, ``X-Swift-Cc`` and ``X-Swift-Bcc`` headers.
 
+delivery_whitelist
+~~~~~~~~~~~~~~~~~~
+
+**type**: ``array``
+
+Used in combination with ``delivery_address``. If set, emails matching any of these
+patterns will be delivered like normal, instead of being sent to `delivery_address`.
+For details, see `the Cookbook entry.`_
```
Comment (xabbuh) [inline reference/configuration/swiftmailer.rst]: Please use the `ref` role for internal links like this:

```rst
:ref:`the cookbook entry <sending-to-a-specified-address-but-with-exceptions>`
```

Comment (TerjeBr) [inline reference/configuration/swiftmailer.rst]: OK, changed that.

**Comment by xabbuh** [comment]
@TerjeBr Can you please also add the label in `/cookbook/email/dev_environment.rst`?

It should look like this (around line 122):

```rst
.. _sending-to-a-specified-address-but-with-exceptions:

Sending to a Specified Address but with Exceptions
..................................................
```

File: reference/configuration/swiftmailer.rst [hunk 3]
```diff
@@ -156,6 +157,15 @@ emails sent during development go to a single account.
 This uses ``Swift_Plugins_RedirectingPlugin``. Original recipients are available
 on the ``X-Swift-To``, ``X-Swift-Cc`` and ``X-Swift-Bcc`` headers.
 
+delivery_whitelist
+~~~~~~~~~~~~~~~~~~
+
+**type**: ``array``
+
+Used in combination with ``delivery_address``. If set, emails matching any of these
+patterns will be delivered like normal, instead of being sent to `delivery_address`.
```
Comment (WouterJ) [inline reference/configuration/swiftmailer.rst]: sorry, one last thing: you have to use double backticks for inline codeblocks in reStructured text.

Comment (TerjeBr) [inline reference/configuration/swiftmailer.rst]: ok, done that.

**Comment by WouterJ** [comment]
Looking good, thanks! :+1:

**Comment by xabbuh** [comment]
:+1: I like it.

And a huge thank you to @e-weimann for the initial work on this.

**Comment by TerjeBr** [comment]
@e-weimann opened #4925 just after I had filed #4924 . I did the initial work of documenting this feature, and I also provided @e-weimann with the text for the reference update.

But I guess we can thank @e-weimann for taking the initiative to help us not letting the reference section be forgotten on this issue.

**Comment by e-weimann** [comment]
@TerjeBr It is her PR, not his and it was coincidence that I opened the pull request after #4924. I found the option in the source, but couldn't find information about it in the documentation. Thanks for finishing the this.

**Comment by weaverryan** [comment]
Thanks very much to both of you (@TerjeBr and @e-weimann) for making sure this cool little feature got its docs. Like I said on the original PR, I didn't even know this existed! Cheers!

**Pull request merged**